packages/smartcontracts/src/tests/BridgeProxy.evmToDefi.spec.ts

```
@@ -65,5 +65,79 @@ describe('EVM --> DeFiChain', () => {
+        }),
+      ).to.be.revertedWithCustomError(proxyBridge, 'REQUESTED_AMOUNT_IS_ZERO');
+    });
+
+    it('Successfully bridging', async () => {
+      const { proxyBridge, communityAddress } = await loadFixture(deployContracts);
+      await proxyBridge.addSupportedTokens(ethers.constants.AddressZero, toWei('20'));
+      // Contract address balance should be zero
+      expect(await ethers.provider.getBalance(proxyBridge.address)).to.equal(0);
+      // Need to add to the timestamp of the previous block to match the next block the tx is mined in
+      const expectedTimestamp = (await time.latest()) + 1;
+      // Checking contract balance: should be 10 - txFee test tokens
+      const txFee = await proxyBridge.transactionFee();
+      const amount = toWei('10');
+      const expectedBalance = amountAfterFee({ amount, transactionFee: txFee });
+      const communityAddrBalanceBeforeBridge = await ethers.provider.getBalance(communityAddress);
+      // Bridging 10 tokens
+      await expect(
+        proxyBridge.bridgeToDeFiChain(ethers.constants.AddressZero, ethers.constants.AddressZero, 0, {
+          value: amount,
+        }),
+      )
+        .to.emit(proxyBridge, 'BRIDGE_TO_DEFI_CHAIN')
+        .withArgs(ethers.constants.AddressZero, ethers.constants.AddressZero, expectedBalance, expectedTimestamp);
+
+      expect(await ethers.provider.getBalance(proxyBridge.address)).to.equal(expectedBalance);
+      // Checking the community wallet balance
+      const communityAddrBalanceAfterBridge = await ethers.provider.getBalance(communityAddress);
+      expect(communityAddrBalanceAfterBridge).to.equal(
+        communityAddrBalanceBeforeBridge.add(amount.sub(expectedBalance)),
+      );
+    });
+
+    it('Successfully revert when _amount parameter > 0', async () => {
+      const { proxyBridge } = await loadFixture(deployContracts);
+      await proxyBridge.addSupportedTokens(ethers.constants.AddressZero, toWei('20'));
+      await expect(
```

Quick comment, when bridging ETH. `_amount` value should be neglected anyway. Do we really need to throw this error?

---

Yes, it does not matter in terms of logic, but feel it is better this way for the user experience

---

My guess is that it's just a sanity check? I think it's fine 🤔 , although yes it technically doesn't have any impact whether or not it's 0